package.json

```diff
@@ -20,12 +22,15 @@
   },
   "homepage": "https://github.com/opentable/hapi-hobknob-client",
   "devDependencies": {
-    "grunt": "^0.4.5",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-jshint": "^0.10.0",
-    "grunt-mocha-test": "^0.11.0",
-    "should": "^4.0.4"
+    "eslint": "^3.3.1",
+    "eslint-config-defaults": "^9.0.0",
+    "expect.js": "^0.3.1",
+    "istanbul": "^0.4.5",
+    "mocha": "^3.0.2",
+    "rewire": "^2.5.2"
   },
   "dependencies": {
+    "hobknob-client-nodejs": "^2.0.0",
+    "lodash": "^4.17.2"
```

Comment:
there is only one usage of lodash in the entire project, kind of overkill to pull it in IMHO